packages/components/src/navigation/context.js

```diff
@@ -18,5 +18,18 @@ export const NavigationContext = createContext( {
 	activeMenu: ROOT_MENU,
 	setActiveItem: noop,
 	setActiveMenu: noop,
+
+	navigationTree: {
+		getItem: noop,
+		getMenu: noop,
+
+		items: {},
+		addItem: noop,
+		removeItem: noop,
+
+		menus: {},
+		addMenu: noop,
+		removeMenu: noop,
+	},
```

Comment:
Nitpick: I'm personally a fan of alphabetical order, but this looks good to me anyway.
Just, would you mind group the two `get` with their corresponding types?
E.g.
```
items: {},
getItem: noop,
addItem: noop,
removeItem: noop,
```

See https://github.com/WordPress/gutenberg/pull/25340/files#diff-60d836aca901f5444058a3ebac6e588fR21-R31

Reply:
Done